tests/components/filter_menu.test.ts

```diff
@@ -187,6 +187,41 @@ describe("Filter menu component", () => {
       ]);
     });
 
+    test("Clear all work on the displayed values", async () => {
+      await openFilterMenu();
+      const searchInput = fixture.querySelector(".o-filter-menu input") as HTMLInputElement;
+      searchInput.value = "1";
+      searchInput.dispatchEvent(new Event("input", { bubbles: true }));
```

Comment:
you can use `setInputValueAndTrigger` :wink: (and maybe it needs to be adapted for bubbles: true)
```suggestion
      setInputValueAndTrigger(".o-filter-menu input", "1", "input");
```